climakitae/data_loaders.py

```diff
@@ -358,14 +406,27 @@ def _get_data_one_var(selections, location, cat):
 
     # Perform subsetting operations
     for dname, dset in data_dict.items():
-        # Add simulation as a coord
-        dset = dset.assign_coords({"simulation": dset.attrs["source_id"]})
+        # Break down name into component parts
+        (
+            activity_id,
+            institution_id,
+            source_id,
+            experiment_id,
+            table_id,
+            grid_label,
+        ) = dname.split(".")
```

Comment:
Shouldn't this stuff be in the metadata, or how the catalog is organized, so we don't need to be parsing strings?

Reply:
It is in the catalog organization actually. The dname is assigned by intake upon retrieval using information from the catalog. This was just an easy way to access the catalog info without needing to dig into the data attributes, which are inconsistent across LOCA/WRF/etc.

Reply:
Okay, this can also be a separate trello card, but I'd rather future-proof other data being added that doesn't follow the same file naming convention. If the same attribute goes by two different names we can have a special function to handle that perhaps that references some look-up, or harmonize them in the catalog itself.

Reply:
The LOCA data doesn't have these attributes unfortunately, it only has units and long_name if I remember correctly.

Reply:
> * I seem to recall that CESM2 doesn't have LOCA2 daily min and max temperature, but it _should_ have specific humidity, yet I don't see it among the simulations even when I select specific humidity

Yep, something is up with the CESM2. I must have introduced a bug somewhere that drops it. I'll look into that now, thanks for pointing it out